contrib/apiage.py

```diff
@@ -286,6 +354,10 @@ def main():
 
     if cli.current_tag:
         tag_to_versions(cli, cli.current_tag)
+    elif cli.placeholder_versions:
```

Comment:
I know it doesn't make sense to have `--current-tag` along with `--placeholder-versions`. But still can we use `if` instead of `elif`?

Reply:
Yeah, its about precedence. I think its fine then.